src/plugins/procdump/procdump.cpp

```diff
@@ -853,12 +855,19 @@ static event_response_t terminate_process_cb(drakvuf_t drakvuf,
     if (it != plugin->terminating.end())
     {
         if (!it->second)
+        {
+            // TODO Check if this line could be reached (look "detach" function)
             plugin->terminating.erase(info->attached_proc_data.pid);
+            plugin->terminated_processes[info->attached_proc_data.pid] = true;
```

Comment:
Code consistency + it's best to avoid `operator[]` on `std::map`.
```suggestion
            plugin->terminated_processes.at(info->attached_proc_data.pid) = true;
```

Reply:
Have switched to `std::shared_ptr<const...`